hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/embedded/EmbeddedTimelineService.java

```diff
@@ -84,7 +88,10 @@ private FileSystemViewManager createViewManager() {
   }
 
   public void startServer() throws IOException {
-    server = new TimelineService(preferredPort, viewManager, hadoopConf.newCopy(), numThreads, shouldCompressOutput, useAsync);
+    server = new TimelineService(preferredPort, viewManager,
+        FSUtils.getFs(basePath, hadoopConf.newCopy()), hadoopConf.newCopy(), numThreads, shouldCompressOutput, useAsync,
```

Comment:
I cannot use `HoodieWriteConfig` instance in the `TimelineService` class since `hudi-client-common` module, which has the `HoodieWriteConfig`, depends on `hudi-timeline-service` module with `TimelineService` class.  Otherwise, there will be a cyclic module dependency.  I instead use the `TimeService.Confg` instance to pass the configs.